bricks/net/http/headers/headers.h

```diff
@@ -224,6 +224,12 @@ struct Headers final {
     return *this;
   }
 
+  Headers& Clear(const std::string& header) {
```

Comment:
Clear -> Remove? Clear usually stands for RemoveAll and goes without arguments

Reply:
3d8b2cc9